crawl-ref/source/traps.cc

```diff
@@ -457,19 +457,30 @@ vector<coord_def> find_golubria_on_level()
     return ret;
 }
 
-static bool _find_other_passage_side(coord_def& to)
+enum _passage_type
```

Comment:
Neither the enum nor the enum values should have leading `_`s. We just use those for names with static linkage.